### ISSUE TITLE
Webshare: 'NoneType' object has no attribute 'get_info'

### ISSUE THREAD

**Issue by BKPepe** [pyload/pyload#2858]
```
Jun   4 01:41:21 Debian-NAS pyLoadCore.py[10354]: 04.06.2017 01:41:21 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://webshare.cz/file/10q5Ot1m4s/fstifofruyys6yo9vvqvdzf7kxxsye-hawk-part06-rar (pid:25)
Jun  4 01:41:21 Debian-NAS pyLoadCore.py[10354]: 04.06.2017 01:41:21 INFO      ADDON UnSkipOnFail: No duplicates found
Jun  4 01:41:21 Debian-NAS pyLoadCore.py[10354]: 04.06.2017 01:41:21 DEBUG     ADDON ExternalScripts: No script found under folder `download_failed`
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: 04.06.2017 01:41:23 INFO      Download starts: https://webshare.cz/file/5pl51m5LRQ/fstifofruyys6yo9vvqvdzf7kxxsye-hawk-part07-rar
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: 04.06.2017 01:41:23 DEBUG     ADDON ExternalScripts: No script found under folder `download_preparing`
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: 04.06.2017 01:41:23 DEBUG     ADDON UserAgentSwitcher: Setting connection timeout to 60 seconds
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: 04.06.2017 01:41:23 DEBUG     ADDON UserAgentSwitcher: Setting maximum redirections to 10
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: 04.06.2017 01:41:23 DEBUG     ADDON UserAgentSwitcher: Use custom user-agent string `Mozilla/5.0 (Windows NT 6.1; WOW64; rv:38.0) Gecko/20100101 Firefox/38.0`
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: 04.06.2017 01:41:23 DEBUG     HOSTER WebshareCz[190]: Plugin version: 0.25
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: 04.06.2017 01:41:23 DEBUG     HOSTER WebshareCz[190]: Plugin status: testing
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: 04.06.2017 01:41:23 WARNING   HOSTER WebshareCz[190]: Plugin may be unstable
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: 04.06.2017 01:41:23 DEBUG     ADDON ExternalScripts: No script found under folder `download_processed`
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: 04.06.2017 01:41:23 WARNING   Download failed: https://webshare.cz/file/5pl51m5LRQ/fstifofruyys6yo9vvqvdzf7kxxsye-hawk-part07-rar | 'NoneType' object has no attribute 'get_info'
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: Traceback (most recent call last):
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: File "/home/pepe/pyload/module/PluginThread.py", line 187, in run
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: pyfile.plugin.preprocessing(self)
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: File "/home/pepe/pyload/module/plugins/internal/Base.py", line 294, in preprocessing
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: return self._process(*args, **kwargs)
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: File "/home/pepe/pyload/module/plugins/internal/Hoster.py", line 102, in _process
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: self._setup()
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: File "/home/pepe/pyload/module/plugins/internal/Base.py", line 144, in _setup
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: self.load_account()  # @TODO: Move to PluginThread in 0.4.10
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: File "/home/pepe/pyload/module/plugins/internal/Hoster.py", line 94, in load_account
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: Base.load_account(self)
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: File "/home/pepe/pyload/module/plugins/internal/Base.py", line 180, in load_account
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: self.account.choose()
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: File "/home/pepe/pyload/module/plugins/internal/misc.py", line 228, in new
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: return fn(*args, **kwargs)
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: File "/home/pepe/pyload/module/plugins/internal/Account.py", line 408, in choose
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: user = self.select()[0]
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: File "/home/pepe/pyload/module/plugins/internal/misc.py", line 228, in new
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: return fn(*args, **kwargs)
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: File "/home/pepe/pyload/module/plugins/internal/Account.py", line 349, in select
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: info = self.accounts[user]['plugin'].get_info()
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: AttributeError: 'NoneType' object has no attribute 'get_info'
Jun  4 01:41:23 Debian-NAS pyLoadCore.py[10354]: 04.06.2017 01:41:23 INFO      Debug Report written to debug_WebshareCz_04-06-2017_01-41-23.zip
```
I have 47 files of debug. If you want more logs I can help you with it.
[debug_WebshareCz_04-06-2017_01-41-23.zip](https://github.com/pyload/pyload/files/1050229/debug_WebshareCz_04-06-2017_01-41-23.zip)

**Comment by GammaC0de** [comment]
WebshareC account plugin requires python `hashlib` module to be installed..

**Comment by BKPepe** [comment]
I will check it. But it should be mention somewhere. 
Wheh I added account for Webshare it says that I need to have passlib.

**Comment by GammaC0de** [comment]
OK, so it is defensively the problem.

I agree that the plugin could be improved but I did not write it, lets leave it for later and focus on your problem first..

**Comment by BKPepe** [comment]
It's weird, because it isn't happening with every file.

I'm not sure if I have installed hashlib anyway I tried to install python-pip and did this:
pip install hashlib ; result here:  https://pastebin.com/1aY5iD5V
Googled and find that I should use easy_install hashlib; result:  https://pastebin.com/wSR2mLdZ

It's quite old and I couldn't find related info. When I will find something that I will edit my comment.

**Comment by GammaC0de** [comment]
Sorry, my bad, it's `passlib` not `hashlib`

**Comment by BKPepe** [comment]
That's what I have. I installed it immediately after I entered account for it.
For few downloads it was OK, but for few it wasn't.

**Comment by GammaC0de** [comment]
OK, so enable DEBUG mode and see if there is something helpfull

**Comment by BKPepe** [comment]
Man., I had enable debug and you can see attachment which I included and I have enabled debug (still)

**Comment by GammaC0de** [comment]
Do you still get the `ERROR Error importing WebshareCz: No module named passlib.hash` error?

**Comment by BKPepe** [comment]
No. It was only when I tried to add account.

**Comment by GammaC0de** [comment]
Restart pyload and see if is shows again.
If so, the account is not working.

**Comment by BKPepe** [comment]
Honestly, I need to tell you that you're having a bad day.
As I told you earlier. I can't reproduce it, yet. (waiting for this fault)

After I installed passlib I don't restarted pyload. It wasn't necessary.
I have premium on my account, so I'm using premium download.

But this bug happened after #2857 , when I enabled debug and I don't even mention that I'm trying premium download, because my credentials (my name and mail) are included in debug attachment archive, which I included and they was generated by PyLoad.
So in the end I have restarted pyload, when this bug appears.

I really doubt that my account wasn't working. You're asking questions, which are obvious.
You could tell me that you want complete log, where you can see everything.

**Comment by GammaC0de** [comment]
I am closing all your issues except #2859, all the issues you have opened are related.

**Comment by BKPepe** [comment]
Hello,
It happened again. Many times. See:
https://ctrlv.cz/lCXb
Here is log from daemon.log

```
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:55 INFO      Download restarted: PRHHDMACZ_E.part19.rar (2) | Html file
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:55 INFO      Download starts: PRHHDMACZ_E.part19.rar (2)
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:55 DEBUG     ADDON ExternalScripts: No script found under folder `download_preparing`
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:55 DEBUG     ADDON UserAgentSwitcher: Setting connection timeout to 60 seconds
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:55 DEBUG     ADDON UserAgentSwitcher: Setting maximum redirections to 10
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:55 DEBUG     ADDON UserAgentSwitcher: Use custom user-agent string `Mozilla/5.0 (Windows NT 6.1; WOW64; rv:38.0) Gecko/20100101 Firefox/38.0`
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:55 DEBUG     HOSTER WebshareCz[203]: Plugin version: 0.25
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:55 DEBUG     HOSTER WebshareCz[203]: Plugin status: testing
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:55 WARNING   HOSTER WebshareCz[203]: Plugin may be unstable
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:55 DEBUG     ADDON ExternalScripts: No script found under folder `download_processed`
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:55 DEBUG     ADDON ExternalScripts: No script found under folder `package_processed`
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:55 DEBUG     ADDON ExternalScripts: No script found under folder `package_failed`
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:55 WARNING   Download failed: PRHHDMACZ_E.part19.rar (2) | 'NoneType' object has no attribute 'get_info'
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: Traceback (most recent call last):
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/PluginThread.py", line 187, in run
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: pyfile.plugin.preprocessing(self)
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Base.py", line 294, in preprocessing
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: return self._process(*args, **kwargs)
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Hoster.py", line 102, in _process
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: self._setup()
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Base.py", line 144, in _setup
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: self.load_account()  # @TODO: Move to PluginThread in 0.4.10
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Hoster.py", line 94, in load_account
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: Base.load_account(self)
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Base.py", line 180, in load_account
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: self.account.choose()
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/misc.py", line 228, in new
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: return fn(*args, **kwargs)
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Account.py", line 408, in choose
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: user = self.select()[0]
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/misc.py", line 228, in new
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: return fn(*args, **kwargs)
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Account.py", line 349, in select
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: info = self.accounts[user]['plugin'].get_info()
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: AttributeError: 'NoneType' object has no attribute 'get_info'
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:55 INFO      Debug Report written to debug_WebshareCz_10-06-2017_00-53-55.zip
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:55 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: PRHHDMACZ_E.part19.rar (2) (pid:29)
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:55 INFO      ADDON UnSkipOnFail: No duplicates found
Jun 10 00:53:55 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:55 DEBUG     ADDON ExternalScripts: No script found under folder `download_failed`
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:57 INFO      Download starts: https://webshare.cz/#/file/4Zw36d71i6/thgrm07-part1-rar
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:57 DEBUG     ADDON ExternalScripts: No script found under folder `download_preparing`
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:57 DEBUG     ADDON UserAgentSwitcher: Setting connection timeout to 60 seconds
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:57 DEBUG     ADDON UserAgentSwitcher: Setting maximum redirections to 10
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:57 DEBUG     ADDON UserAgentSwitcher: Use custom user-agent string `Mozilla/5.0 (Windows NT 6.1; WOW64; rv:38.0) Gecko/20100101 Firefox/38.0`
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:57 DEBUG     HOSTER WebshareCz[224]: Plugin version: 0.25
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:57 DEBUG     HOSTER WebshareCz[224]: Plugin status: testing
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:57 WARNING   HOSTER WebshareCz[224]: Plugin may be unstable
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:57 DEBUG     ADDON ExternalScripts: No script found under folder `download_processed`
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:57 WARNING   Download failed: https://webshare.cz/#/file/4Zw36d71i6/thgrm07-part1-rar | 'NoneType' object has no attribute 'get_info'
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: Traceback (most recent call last):
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/PluginThread.py", line 187, in run
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: pyfile.plugin.preprocessing(self)
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Base.py", line 294, in preprocessing
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: return self._process(*args, **kwargs)
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Hoster.py", line 102, in _process
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: self._setup()
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Base.py", line 144, in _setup
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: self.load_account()  # @TODO: Move to PluginThread in 0.4.10
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Hoster.py", line 94, in load_account
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: Base.load_account(self)
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Base.py", line 180, in load_account
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: self.account.choose()
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/misc.py", line 228, in new
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: return fn(*args, **kwargs)
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Account.py", line 408, in choose
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: user = self.select()[0]
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/misc.py", line 228, in new
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: return fn(*args, **kwargs)
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Account.py", line 349, in select
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: info = self.accounts[user]['plugin'].get_info()
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: AttributeError: 'NoneType' object has no attribute 'get_info'
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:57 INFO      Debug Report written to debug_WebshareCz_10-06-2017_00-53-57.zip
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:57 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://webshare.cz/#/file/4Zw36d71i6/thgrm07-part1-rar (pid:34)
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:57 INFO      ADDON UnSkipOnFail: No duplicates found
Jun 10 00:53:57 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:57 DEBUG     ADDON ExternalScripts: No script found under folder `download_failed`
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:59 INFO      Download starts: https://webshare.cz/#/file/71U7Z3df3j/thgrm07-part2-rar
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:59 DEBUG     ADDON ExternalScripts: No script found under folder `download_preparing`
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:59 DEBUG     ADDON UserAgentSwitcher: Setting connection timeout to 60 seconds
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:59 DEBUG     ADDON UserAgentSwitcher: Setting maximum redirections to 10
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:59 DEBUG     ADDON UserAgentSwitcher: Use custom user-agent string `Mozilla/5.0 (Windows NT 6.1; WOW64; rv:38.0) Gecko/20100101 Firefox/38.0`
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:59 DEBUG     HOSTER WebshareCz[225]: Plugin version: 0.25
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:59 DEBUG     HOSTER WebshareCz[225]: Plugin status: testing
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:59 WARNING   HOSTER WebshareCz[225]: Plugin may be unstable
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:59 DEBUG     ADDON ExternalScripts: No script found under folder `download_processed`
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:59 WARNING   Download failed: https://webshare.cz/#/file/71U7Z3df3j/thgrm07-part2-rar | 'NoneType' object has no attribute 'get_info'
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: Traceback (most recent call last):
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/PluginThread.py", line 187, in run
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: pyfile.plugin.preprocessing(self)
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Base.py", line 294, in preprocessing
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: return self._process(*args, **kwargs)
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Hoster.py", line 102, in _process
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: self._setup()
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Base.py", line 144, in _setup
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: self.load_account()  # @TODO: Move to PluginThread in 0.4.10
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Hoster.py", line 94, in load_account
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: Base.load_account(self)
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Base.py", line 180, in load_account
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: self.account.choose()
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/misc.py", line 228, in new
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: return fn(*args, **kwargs)
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Account.py", line 408, in choose
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: user = self.select()[0]
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/misc.py", line 228, in new
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: return fn(*args, **kwargs)
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Account.py", line 349, in select
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: info = self.accounts[user]['plugin'].get_info()
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: AttributeError: 'NoneType' object has no attribute 'get_info'
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:59 INFO      Debug Report written to debug_WebshareCz_10-06-2017_00-53-59.zip
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:59 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://webshare.cz/#/file/71U7Z3df3j/thgrm07-part2-rar (pid:34)
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:59 INFO      ADDON UnSkipOnFail: No duplicates found
Jun 10 00:53:59 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:53:59 DEBUG     ADDON ExternalScripts: No script found under folder `download_failed`
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:01 INFO      Download starts: https://webshare.cz/#/file/4al2s332D2/thgrm07-part3-rar
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:01 DEBUG     ADDON ExternalScripts: No script found under folder `download_preparing`
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:01 DEBUG     ADDON UserAgentSwitcher: Setting connection timeout to 60 seconds
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:01 DEBUG     ADDON UserAgentSwitcher: Setting maximum redirections to 10
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:01 DEBUG     ADDON UserAgentSwitcher: Use custom user-agent string `Mozilla/5.0 (Windows NT 6.1; WOW64; rv:38.0) Gecko/20100101 Firefox/38.0`
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:01 DEBUG     HOSTER WebshareCz[226]: Plugin version: 0.25
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:01 DEBUG     HOSTER WebshareCz[226]: Plugin status: testing
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:01 WARNING   HOSTER WebshareCz[226]: Plugin may be unstable
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:01 DEBUG     ADDON ExternalScripts: No script found under folder `download_processed`
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:01 WARNING   Download failed: https://webshare.cz/#/file/4al2s332D2/thgrm07-part3-rar | 'NoneType' object has no attribute 'get_info'
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: Traceback (most recent call last):
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/PluginThread.py", line 187, in run
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: pyfile.plugin.preprocessing(self)
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Base.py", line 294, in preprocessing
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: return self._process(*args, **kwargs)
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Hoster.py", line 102, in _process
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: self._setup()
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Base.py", line 144, in _setup
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: self.load_account()  # @TODO: Move to PluginThread in 0.4.10
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Hoster.py", line 94, in load_account
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: Base.load_account(self)
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Base.py", line 180, in load_account
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: self.account.choose()
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/misc.py", line 228, in new
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: return fn(*args, **kwargs)
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Account.py", line 408, in choose
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: user = self.select()[0]
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/misc.py", line 228, in new
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: return fn(*args, **kwargs)
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Account.py", line 349, in select
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: info = self.accounts[user]['plugin'].get_info()
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: AttributeError: 'NoneType' object has no attribute 'get_info'
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:01 INFO      Debug Report written to debug_WebshareCz_10-06-2017_00-54-01.zip
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:01 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://webshare.cz/#/file/4al2s332D2/thgrm07-part3-rar (pid:34)
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:01 INFO      ADDON UnSkipOnFail: No duplicates found
Jun 10 00:54:01 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:01 DEBUG     ADDON ExternalScripts: No script found under folder `download_failed`
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:03 INFO      Download starts: https://webshare.cz/#/file/3ui4us1UL7/thgrm07-part4-rar
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:03 DEBUG     ADDON ExternalScripts: No script found under folder `download_preparing`
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:03 DEBUG     ADDON UserAgentSwitcher: Setting connection timeout to 60 seconds
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:03 DEBUG     ADDON UserAgentSwitcher: Setting maximum redirections to 10
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:03 DEBUG     ADDON UserAgentSwitcher: Use custom user-agent string `Mozilla/5.0 (Windows NT 6.1; WOW64; rv:38.0) Gecko/20100101 Firefox/38.0`
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:03 DEBUG     HOSTER WebshareCz[227]: Plugin version: 0.25
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:03 DEBUG     HOSTER WebshareCz[227]: Plugin status: testing
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:03 WARNING   HOSTER WebshareCz[227]: Plugin may be unstable
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:03 DEBUG     ADDON ExternalScripts: No script found under folder `download_processed`
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:03 DEBUG     ADDON ExternalScripts: No script found under folder `package_processed`
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:03 DEBUG     ADDON ExternalScripts: No script found under folder `package_failed`
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:03 WARNING   Download failed: https://webshare.cz/#/file/3ui4us1UL7/thgrm07-part4-rar | 'NoneType' object has no attribute 'get_info'
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: Traceback (most recent call last):
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/PluginThread.py", line 187, in run
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: pyfile.plugin.preprocessing(self)
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Base.py", line 294, in preprocessing
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: return self._process(*args, **kwargs)
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Hoster.py", line 102, in _process
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: self._setup()
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Base.py", line 144, in _setup
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: self.load_account()  # @TODO: Move to PluginThread in 0.4.10
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Hoster.py", line 94, in load_account
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: Base.load_account(self)
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Base.py", line 180, in load_account
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: self.account.choose()
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/misc.py", line 228, in new
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: return fn(*args, **kwargs)
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Account.py", line 408, in choose
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: user = self.select()[0]
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/misc.py", line 228, in new
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: return fn(*args, **kwargs)
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Account.py", line 349, in select
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: info = self.accounts[user]['plugin'].get_info()
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: AttributeError: 'NoneType' object has no attribute 'get_info'
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:03 INFO      Debug Report written to debug_WebshareCz_10-06-2017_00-54-03.zip
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:03 INFO      ADDON UnSkipOnFail: Looking for skipped duplicates of: https://webshare.cz/#/file/3ui4us1UL7/thgrm07-part4-rar (pid:34)
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:03 INFO      ADDON UnSkipOnFail: No duplicates found
Jun 10 00:54:03 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:03 DEBUG     ADDON ExternalScripts: No script found under folder `download_failed`
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:05 INFO      Download starts: https://webshare.cz/file/3va69o7fCh/whnth16-part1-rar
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:05 DEBUG     ADDON ExternalScripts: No script found under folder `download_preparing`
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:05 DEBUG     ADDON UserAgentSwitcher: Setting connection timeout to 60 seconds
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:05 DEBUG     ADDON UserAgentSwitcher: Setting maximum redirections to 10
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:05 DEBUG     ADDON UserAgentSwitcher: Use custom user-agent string `Mozilla/5.0 (Windows NT 6.1; WOW64; rv:38.0) Gecko/20100101 Firefox/38.0`
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:05 DEBUG     HOSTER WebshareCz[228]: Plugin version: 0.25
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:05 DEBUG     HOSTER WebshareCz[228]: Plugin status: testing
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:05 WARNING   HOSTER WebshareCz[228]: Plugin may be unstable
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:05 DEBUG     ADDON ExternalScripts: No script found under folder `download_processed`
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:05 WARNING   Download failed: https://webshare.cz/file/3va69o7fCh/whnth16-part1-rar | 'NoneType' object has no attribute 'get_info'
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: Traceback (most recent call last):
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/PluginThread.py", line 187, in run
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: pyfile.plugin.preprocessing(self)
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Base.py", line 294, in preprocessing
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: return self._process(*args, **kwargs)
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Hoster.py", line 102, in _process
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: self._setup()
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Base.py", line 144, in _setup
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: self.load_account()  # @TODO: Move to PluginThread in 0.4.10
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Hoster.py", line 94, in load_account
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: Base.load_account(self)
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Base.py", line 180, in load_account
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: self.account.choose()
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/misc.py", line 228, in new
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: return fn(*args, **kwargs)
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Account.py", line 408, in choose
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: user = self.select()[0]
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/misc.py", line 228, in new
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: return fn(*args, **kwargs)
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Account.py", line 349, in select
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: info = self.accounts[user]['plugin'].get_info()
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: AttributeError: 'NoneType' object has no attribute 'get_info'
Jun 10 00:54:05 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:54:05 INFO      Debug Report written to debug_WebshareCz_10-06-2017_00-54-05.zip
[debug_UptoboxCom_10-06-2017_00-52-50.zip](https://github.com/pyload/pyload/files/1065696/debug_UptoboxCom_10-06-2017_00-52-50.zip)
[debug_WebshareCz_10-06-2017_00-54-57.zip](https://github.com/pyload/pyload/files/1065694/debug_WebshareCz_10-06-2017_00-54-57.zip)
[debug_WebshareCz_10-06-2017_00-55-42.zip](https://github.com/pyload/pyload/files/1065695/debug_WebshareCz_10-06-2017_00-55-42.zip)
[debug_WebshareCz_10-06-2017_00-56-59.zip](https://github.com/pyload/pyload/files/1065697/debug_WebshareCz_10-06-2017_00-56-59.zip)
[debug_WebshareCz_10-06-2017_00-57-47.zip](https://github.com/pyload/pyload/files/1065698/debug_WebshareCz_10-06-2017_00-57-47.zip)







```


Also it happens for Uptobox.com
```
Jun 10 00:52:27 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:52:27 WARNING   Download failed: https://uptobox.com/yuye1eadjwu7/Aftermath.2017.1080.WEB-DL.DD5.1.H264.CZsub-FGT.mkv | 'NoneType' object has no attribute 'get_info'
Jun 10 00:52:27 Debian-NAS pyLoadCore.py[15819]: Traceback (most recent call last):
Jun 10 00:52:27 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/PluginThread.py", line 187, in run
Jun 10 00:52:27 Debian-NAS pyLoadCore.py[15819]: pyfile.plugin.preprocessing(self)
Jun 10 00:52:27 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Base.py", line 294, in preprocessing
Jun 10 00:52:27 Debian-NAS pyLoadCore.py[15819]: return self._process(*args, **kwargs)
Jun 10 00:52:27 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Hoster.py", line 102, in _process
Jun 10 00:52:27 Debian-NAS pyLoadCore.py[15819]: self._setup()
Jun 10 00:52:27 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Base.py", line 144, in _setup
Jun 10 00:52:27 Debian-NAS pyLoadCore.py[15819]: self.load_account()  # @TODO: Move to PluginThread in 0.4.10
Jun 10 00:52:27 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Hoster.py", line 94, in load_account
Jun 10 00:52:27 Debian-NAS pyLoadCore.py[15819]: Base.load_account(self)
Jun 10 00:52:27 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Base.py", line 180, in load_account
Jun 10 00:52:27 Debian-NAS pyLoadCore.py[15819]: self.account.choose()
Jun 10 00:52:27 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/misc.py", line 228, in new
Jun 10 00:52:27 Debian-NAS pyLoadCore.py[15819]: return fn(*args, **kwargs)
Jun 10 00:52:27 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Account.py", line 408, in choose
Jun 10 00:52:27 Debian-NAS pyLoadCore.py[15819]: user = self.select()[0]
Jun 10 00:52:27 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/misc.py", line 228, in new
Jun 10 00:52:27 Debian-NAS pyLoadCore.py[15819]: return fn(*args, **kwargs)
Jun 10 00:52:27 Debian-NAS pyLoadCore.py[15819]: File "/home/pepe/pyload/module/plugins/internal/Account.py", line 349, in select
Jun 10 00:52:27 Debian-NAS pyLoadCore.py[15819]: info = self.accounts[user]['plugin'].get_info()
Jun 10 00:52:27 Debian-NAS pyLoadCore.py[15819]: AttributeError: 'NoneType' object has no attribute 'get_info'
Jun 10 00:52:27 Debian-NAS pyLoadCore.py[15819]: 10.06.2017 00:52:27 INFO      Debug Report written to debug_UptoboxCom_10-06-2017_00-52-27.zip
```
